examples/Demo/Shared/Pages/Lab/IssueTester.razor

```diff
@@ -1 +1,18 @@
-﻿@page "/IssueTester"
\ No newline at end of file
+﻿@page "/IssueTester"
```

Comment:
This page is probably a "temp" work? :-)

Reply:
Yup, cleared it out